packages/amplify_authenticator/lib/src/widgets/form.dart

```diff
@@ -102,15 +109,38 @@ class AuthenticatorFormState<T extends AuthenticatorForm<T>>
     );
   }
 
+  List<AuthenticatorFormField> get allFields {
+    final fields = HashSet<AuthenticatorFormField>(
+      equals: (a, b) {
+        return a.runtimeType == b.runtimeType && a.field == b.field;
+      },
+      hashCode: (field) => hashValues(field.runtimeType, field.field),
+    );
+
+    fields.addAll(widget.requiredFields);
+    fields.addAll(runtimeFields(context));
+
+    // Add widget's fields last, since they will contain user's custom fields
+    // and will override the default fields included by runtimeFields.
```

Comment:
`.add()` and `.addAll()` will not change the set if an equal value is already in the set, so by adding customFields last they won't override a value that is already there.

Reply:
Oh yeah, I was thinking about it backwards